src/actions.js

```diff
@@ -172,10 +175,13 @@ function isEmptyValue (value) {
   (_.isObject(value) && Object.keys(value).length === 0) // Check if empty object
 }
 
-function dispatchUpdatedResults (dispatch, results) {
-  const aggregatedResult = aggregateResults(results)
+function dispatchUpdatedResults (dispatch, results, getState) {
```

Comment:
Is `getState` being referenced in this function?

Reply:
Nope. Forgot to take the param off. Will remove